packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -832,11 +866,12 @@ export class DeltaManager
             if (result.done) {
                 break;
             }
-            PerformanceEvent.timedExec(
-                this.logger,
-                { eventName: "GetDeltas_OpProcessing", count: result.value.length},
-                () => callback(result.value),
-                { end: true, cancel: "error" });
+            callback(result.value);
+        }
+
+        this.closeAbortController.signal.onabort = null;
+        if (listenerToClear !== undefined) {
+            this._inbound.off("push", listenerToClear);
```

Comment:
Can the `fetchMessages` or `const result = await stream.read();` lines ever throw? I'm wondering if it's possible for this listener to not be cleared in an error scenario.

Reply:
Yes, they could through. I'll make it more robust